pkg/cmd/pulumi/deployment_settings_config.go

```diff
@@ -70,6 +70,8 @@ func saveProjectStackDeployment(psd *workspace.ProjectStackDeployment, stack bac
 }
 
 func newDeploymentCmd() *cobra.Command {
+	var yes bool
```

Comment:
Nit: can you use a more descriptive name, like `autoConfirmEveryConfirmationPrompt`. 
It feels odd for a variable `yes` to have a value `false` :-)

Reply:
While I agree with this point, we use `yes` everywhere else for basically all other commands, so I'd rather we keep it the same (not ideal). There is some work ahead of us that involves refactoring/cleaning up/extending how we do options, so that might be an opportunity to sort all of this at once then.

Reply:
I am following the same naming as other commands here